packages/data/json/building-types/wt-building-type-analysis.json

```diff
@@ -0,0 +1,53 @@
+{
+  "info": {
+    "title": "Building Type Analysis",
+    "subtitle": "",
+    "description": "Not sure? Narrow down whether your project falls under the specifications of a single- or a multi-unit dwelling.",
+    "startingSectionId": "1_intro",
+    "walkthroughTitle": "FPO - Building Type Analysis"
```

Comment:
What is FPO stand for?

Reply:
I always thought it was "for proof only" but according to Wikipedia that's not it 😂. 

https://en.wikipedia.org/wiki/For_position_only

Either way it's just a note that the text here isn't final.